src/test/java/eu/xenit/gradle/docker/compose/DockerComposeConventionImplTest.java

```diff
@@ -228,5 +238,33 @@ public void testFromProjectWithDockerAlfrescoPlugin() {
         assertEquals(DOCKER_IMAGE_ID, composeSettings.getEnvironment().get("SUB_PROJECT1_DOCKER_IMAGE"));
     }
 
+    @Test
+    public void testFromProjectWithoutEnvironmentVariable(){
```

Comment:
This is already tested in a lot of different cases: https://github.com/xenit-eu/alfresco-docker-gradle-plugin/pull/184/files#diff-7a2b51d094442d693dec1e4bad73b36308671e9f0348c1935223147a6c0e8b13R164-R239

So there is no need to add an additional testcase for this.